packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -1051,29 +1062,26 @@ describe('Table Interactive Column Sizing', () => {
             spec(
                 `${name} updates expected dividers as "divider-active" and "column-active"`,
                 async () => {
+                    const dividerActiveDividers: number[] = [];
+                    const columnActiveDividers: number[] = [];
                     const dividers = Array.from(
                         element.shadowRoot!.querySelectorAll('.column-divider')
                     );
-                    const divider = dividers[value.dividerClickIndex]!;
-                    const dividerRect = divider.getBoundingClientRect();
-                    const mouseDownEvent = new MouseEvent('mousedown', {
-                        clientX: (dividerRect.x + dividerRect.width) / 2,
-                        clientY: (dividerRect.y + dividerRect.height) / 2
-                    });
-                    const mouseUpEvent = new MouseEvent('mouseup');
-                    divider.dispatchEvent(mouseDownEvent);
-                    await waitForUpdatesAsync();
-                    const dividerActiveDividers = [];
-                    const columnActiveDividers = [];
-                    for (let i = 0; i < dividers.length; i++) {
-                        if (dividers[i]!.classList.contains('divider-active')) {
-                            dividerActiveDividers.push(i);
+                    await pageObject.clickAndReleaseColumnDivider(
+                        value.columnIndex,
+                        value.rightDivider,
+                        (): void => {
+                            for (let i = 0; i < dividers.length; i++) {
+                                const classes = dividers[i]!.classList;
+                                if (classes.contains('divider-active')) {
+                                    dividerActiveDividers.push(i);
+                                }
+                                if (classes.contains('column-active')) {
+                                    columnActiveDividers.push(i);
+                                }
+                            }
```

Comment:
This doesn't seem very ergonomic as a pageobject api. It does seem like the test had too many element specific details that were not well encapsulated in the page object, but this isn't encapsulating the details in the page object well either. Are there higher level page object apis we can use to encapsulate this better?